src/pages/Pool-V3/hooks/useCreatePosition.ts

```diff
@@ -65,10 +65,10 @@ const useCreatePosition = (
 
   const xUsd =
     // zapInResponse &&
-    tokenX && ((extendPrices?.[tokenX?.coinGeckoId] * amountX)).toFixed(6);
+    tokenX && (extendPrices?.[tokenX?.coinGeckoId] * amountX).toFixed(6);
```

Comment:
when `tokenX` is existed, you dont need use optional chaining `tokenX?. ` only use `tokenX.`

same comment with tokenY

Reply:
you made my day